src/app/pages/task-calendar/cron/cron-form/cron-form.component.ts

```diff
@@ -61,38 +61,40 @@ export class CronFormComponent implements AfterViewInit {
            tooltip: T('Enter the full path to the command or script to be run.'),
          }, 
          {
-           type: 'select',
+           type: 'combobox',
            name: 'cron_user',
            placeholder: T('Run As User'),
            tooltip: T('Choose a user account to run the <b>Command</b>. The\
              user must have permission to run the <b>Command</b>.'),
-             options: [],
-             required: true,
-             validation : [ Validators.required ],
+           options: [],
+           required: true,
+           validation : [ Validators.required ],
          }, 
          {
            type: 'scheduler',
            name: 'cron_picker',
            placeholder: T('Schedule a Cron Job'),
            tooltip: T('Choose one of the convenient presets\
              or choose <b>Custom</b> to trigger the advanced scheduler UI'),
-             required: true
+           validation: [ Validators.required ],
+           required: true,
+           value: "0 0 * * *"
```

Comment:
if you change `value` to from `0 0 * * *` to `59 23 * * *` , I see that I am not getting `Custom Preset` which it should. I think this should be fixed